amazonredshift_blueprints/upload_file.py

```diff
@@ -251,14 +216,12 @@ def main():
         for index, key_name in enumerate(matching_file_names):
```

Comment:
Not sure if this would work when using insert-method = replace. We should test and if it can't be used with replace we should handle the error

Reply:
Going to test locally to see if this works